R/impute.R

```diff
@@ -0,0 +1,49 @@
+impute <- function(transformed_cont, n_missing) {
```

Comment:
This function needs a roxygen docstring -- see other functions for examples.

https://r-pkgs.org/man.html#roxygen2-basics